doc/_index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
```

Comment:
this looks like puppet-strings docs as html output. We only save the markdown version (REFERENCE.md). Please delte `doc/` and update the REFERENCE.md with `bundle exec rake reference`

Reply:
Done